rsconnect_jupyter/static/connect.js

```diff
@@ -21,6 +22,13 @@ define([
     Canceled: 'canceled'
   };
 
+  var ContentsManager = new Contents.Contents({ base_url: Jupyter.notebook.base_url });
+  var lastSlashInNotebookPath = Jupyter.notebook.notebook_path.lastIndexOf('/');
+  var notebookDirectory = '';
+  if (lastSlashInNotebookPath !== -1) {
+    notebookDirectory = Jupyter.notebook.notebook_path.slice(0, Jupyter.notebook.notebook_path.lastIndexOf('/'));
```

Comment:
nit: this could use `lastSlashInNotebookPath` instead of calling `lastIndexOf` again.